scadalts-ui/src/store/pointHierarchy/index.js

```diff
@@ -26,8 +55,74 @@ const gv = {
 						reject(error);
 					})
 			});
-		}
+		},
+
+		createPointHierarchyNode({ dispatch }, { parentNodeId, nodeName }) {
+			return new Promise((resolve, reject) => {
+				axios.post(`.//pointHierarchy/new/${parentNodeId}/${nodeName}`, requestConfiguration)
+					.then(resp => {
+						resolve(resp.data);
+					})
+					.catch(error => {
+						reject(error);
+					});
+			});
+		},
+
+		editPointHierarchyNode({ dispatch }, { nodeId, parentNodeId, nodeName }) {
+			return new Promise((resolve, reject) => {
+				axios.post(`.//pointHierarchy/edit/${parentNodeId}/${nodeId}/${nodeName}`, requestConfiguration)
+					.then(resp => {
+						resolve(resp.data);
+					})
+					.catch(error => {
+						reject(error);
+					});
+			});
+		},
+
+		movePointHierarchyNode({ dispatch }, { nodeId, parentNodeId, newParentNodeId, isFolder }) {
+			return new Promise((resolve, reject) => {
+				axios.post(`.//pointHierarchy/move/${nodeId}/${parentNodeId}/${newParentNodeId}/${isFolder}`, requestConfiguration)
+					.then(resp => {
+						resolve(resp.data);
+					})
+					.catch(error => {
+						reject(error);
+					});
+			});
+		},
+
+		deletePointHierarchyNode({ dispatch }, { nodeId, parentNodeId, isFolder }) {
+			return new Promise((resolve, reject) => {
+				axios.post(`.//pointHierarchy/del/${parentNodeId}/${nodeId}/${isFolder}`, requestConfiguration)
+					.then(resp => {
+						resolve(resp.data);
+					})
+					.catch(error => {
+						reject(error);
+					});
+			});
+		},
 	},
 	getters: {},
 };
 export default gv;
+
+
+export function searchFolderInHierarchy(array, folderId, debug=false) {
```

Comment:
I think it'd be better to name array param as datapoints and replace for by for of loop